ansible_collections/arista/avd/molecule/eos_validate_state/reports/FABRIC-state.csv

```diff
@@ -1,1711 +1,1711 @@
+51,dc1-leaf1a,Hardware,VerifyTemperature,Verifies the device temperature.,,NOT RUN,
+52,dc1-leaf1a,Hardware,VerifyTemperature,Verifies the device temperature.,,NOT RUN,
+53,dc1-leaf1a,Hardware,VerifyTransceiversManufacturers,Verifies if all transceivers come from approved manufacturers.,,NOT RUN,
+54,dc1-leaf1a,Hardware,VerifyTransceiversManufacturers,Verifies if all transceivers come from approved manufacturers.,"Accepted Manufacturers: 'Arista Networks', 'Arastra, Inc.', 'Not Present'",NOT RUN,
+55,dc1-leaf1a,Hardware,VerifyTransceiversTemperature,Verifies the transceivers temperature.,,NOT RUN,
+56,dc1-leaf1a,Interfaces,VerifyInterfacesStatus,Verifies the status of the provided interfaces.,Interface Ethernet1 - P2P_LINK_TO_DC1-SPINE1_Ethernet1 = 'up',NOT RUN,
+57,dc1-leaf1a,Interfaces,VerifyInterfacesStatus,Verifies the status of the provided interfaces.,Interface Ethernet2 - P2P_LINK_TO_DC1-SPINE2_Ethernet1 = 'up',NOT RUN,
+58,dc1-leaf1a,Interfaces,VerifyInterfacesStatus,Verifies the status of the provided interfaces.,Interface Ethernet3 - MLAG_PEER_dc1-leaf1b_Ethernet3 = 'up',NOT RUN,
+59,dc1-leaf1a,Interfaces,VerifyInterfacesStatus,Verifies the status of the provided interfaces.,Interface Ethernet4 - MLAG_PEER_dc1-leaf1b_Ethernet4 = 'up',NOT RUN,
+60,dc1-leaf1a,Interfaces,VerifyInterfacesStatus,Verifies the status of the provided interfaces.,Interface Ethernet5 - dc1-leaf1-server1_PCI1 = 'up',NOT RUN,
+61,dc1-leaf1a,Interfaces,VerifyInterfacesStatus,Verifies the status of the provided interfaces.,Interface Ethernet6 - P2P_LINK_TO_DC1-WAN1_Ethernet1 = 'up',NOT RUN,
+62,dc1-leaf1a,Interfaces,VerifyInterfacesStatus,Verifies the status of the provided interfaces.,Interface Ethernet7 - P2P_LINK_TO_DC1-WAN2_Ethernet1 = 'up',NOT RUN,
+63,dc1-leaf1a,Interfaces,VerifyInterfacesStatus,Verifies the status of the provided interfaces.,Interface Ethernet8 - DC1-LEAF1C_Ethernet1 = 'up',NOT RUN,
+64,dc1-leaf1a,Interfaces,VerifyInterfacesStatus,Verifies the status of the provided interfaces.,Interface Loopback0 - EVPN_Overlay_Peering = 'up',NOT RUN,
+65,dc1-leaf1a,Interfaces,VerifyInterfacesStatus,Verifies the status of the provided interfaces.,Interface Loopback1 - VTEP_VXLAN_Tunnel_Source = 'up',NOT RUN,
+66,dc1-leaf1a,Interfaces,VerifyInterfacesStatus,Verifies the status of the provided interfaces.,Interface Loopback10 - VRF10_VTEP_DIAGNOSTICS = 'up',NOT RUN,
+67,dc1-leaf1a,Interfaces,VerifyInterfacesStatus,Verifies the status of the provided interfaces.,Interface Loopback11 - VRF11_VTEP_DIAGNOSTICS = 'up',NOT RUN,
+68,dc1-leaf1a,Interfaces,VerifyInterfacesStatus,Verifies the status of the provided interfaces.,Interface Port-Channel3 - MLAG_PEER_dc1-leaf1b_Po3 = 'up',NOT RUN,
+69,dc1-leaf1a,Interfaces,VerifyInterfacesStatus,Verifies the status of the provided interfaces.,Interface Port-Channel5 - dc1-leaf1-server1_PortChannel dc1-leaf1-server1 = 'up',NOT RUN,
```

Comment:
same as above comment

Reply:
Generated from molecule, see the other comment.